articles/api-management/how-to-configure-local-metrics-logs.md

```diff
@@ -190,12 +190,12 @@ Now we have everything deployed and configured, the self-hosted gateway should r
 
 Make some API calls through the self-hosted gateway, if everything is configured correctly, you should be able to view below metrics:
 
-| Metric  | Description |
+| Metric        | Description |
 | ------------- | ------------- |
-| Requests  | Number of API requests in the period |
-| DurationInMS | Number of milliseconds from the moment gateway received request until the moment response sent in full |
-| BackendDurationInMS | Number of milliseconds spent on overall backend IO (connecting, sending and receiving bytes)  |
-| ClientDurationInMS | Number of milliseconds spent on overall client IO (connecting, sending and receiving bytes)  |
+| requests_total  | Number of API requests in the period |
+| request_duration_seconds | Number of milliseconds from the moment gateway received request until the moment response sent in full |
+| request_backend_duration_seconds_count | Number of milliseconds spent on overall backend IO (connecting, sending and receiving bytes)  |
```

Comment:
Just checking that this metric name is "...._count"?

Reply:
No, this should be the following indeed:
```suggestion
| request_backend_duration_seconds | Number of milliseconds spent on overall backend IO (connecting, sending and receiving bytes)  |
```

Reply:
Updated, well-spotted!